### ISSUE TITLE
Ships moving to board other ships will not fire turrets properly

### ISSUE THREAD

**Issue by Bladewood** [endless-sky/endless-sky#2112]
It always comes out to a straight line aimed forward, never at any actual target.

Consider a situation in Kor Sestor & Kor Mereti space where you are initially cloaked. After one side emerges victorious and begins moving to repair damaged ships, you uncloak, and then approach the victors.
They will try to fire at you, but if they are boarding ships, they won't aim their turrets properly.

**Comment by MessyMix** [comment]
@Bladewood, this behavior is also apparent in the player's own actions. If you try and board a ship, and then start firing, your turrets will only point forward. If you target another ship, then it will cancel your boarding action. That is why the AI can only shoot forward when boarding, and I think this is intentional behavior.

**Comment by Alkallid** [comment]
Yes, it was mentioned before, because it would end up destroying your boarding target.
Maybe have a secondary targeting display specifically for boarding targets?

**Comment by MessyMix** [comment]
Too many buttons, and too many targets, imo. We don't want it to be too cluttered.

**Comment by whismerhill** [comment]
I have actually opened this : https://github.com/endless-sky/endless-sky/issues/2081
to mention that changing targets cancel boarding
and that I thought that with 0.9.5 fleet orders, it could be desirable to be allowed to change target
just like you can change target when using jumping autopilot or landing autopilot without interrupting the autopilot.

Of course given what MessyMix just said I suppose other adjustments would have to be made then

**Comment by Bladewood** [comment]
Can we make it, then, so that they will never fire unless a hostile ship is in their direction?

I'm pretty close, but I'm on the other side of them, and they still fire...

**Comment by MessyMix** [comment]
I don't quite understand what you're saying.

**Comment by Nechochwen-D** [comment]
MZ has said before that tracking multiple targets is a potential feature for 1+ but will require a much more efficient AI system than we currently have.

**Comment by Bladewood** [comment]
> 0.9.x

gg

**Comment by tehhowch** [comment]
This may be similar to / related to @KiLEdEnNis 's issue: https://github.com/endless-sky/endless-sky/issues/3149
